com.beamprivacy.desktop.yaml

```diff
@@ -0,0 +1,19 @@
+app-id: com.beamprivacy.desktop
```

Comment:
Is this an actual domain? Project seems to use https://beam.mw/ so it should be `mw.beam.Wallet` – ideally you should actually ask developers.